src/db.rs

```diff
@@ -29,10 +29,17 @@ impl Db {
         Self::default()
     }
 
-    pub fn update_account(&mut self, address: Address, nonce: u64, balance: U256) {
+    pub fn update_account(
```

Comment:
Let's call it `set_account` instead
```suggestion
    pub fn set_account(
```